deepmd/pt/infer/deep_eval.py

```diff
@@ -64,29 +101,55 @@ def __init__(
         else:
             raise TypeError("auto_batch_size should be bool, int, or AutoBatchSize")
 
-    def eval(
-        self,
-        coords: Union[np.ndarray, torch.Tensor],
-        cells: Optional[Union[np.ndarray, torch.Tensor]],
-        atom_types: Union[np.ndarray, torch.Tensor, List[int]],
-        atomic: bool = False,
-    ):
-        raise NotImplementedError
+    def get_rcut(self) -> float:
+        """Get the cutoff radius of this model."""
+        return self.rcut
 
+    def get_ntypes(self) -> int:
+        """Get the number of atom types of this model."""
+        return len(self.type_map)
 
-class DeepPot(DeepEval, DeepPotBase):
-    def __init__(
-        self,
-        model_file: "Path",
-        auto_batch_size: Union[bool, int, AutoBatchSize] = True,
-        neighbor_list=None,
-    ):
-        if neighbor_list is not None:
-            raise NotImplementedError
-        super().__init__(
-            model_file,
-            auto_batch_size=auto_batch_size,
-        )
+    def get_type_map(self) -> List[str]:
+        """Get the type map (element name of the atom types) of this model."""
+        return self.type_map
+
+    def get_dim_fparam(self) -> int:
+        """Get the number (dimension) of frame parameters of this DP."""
+        return 0
+
+    def get_dim_aparam(self) -> int:
+        """Get the number (dimension) of atomic parameters of this DP."""
+        return 0
+
+    @property
+    def model_type(self) -> "DeepEval":
+        """The the evaluator of the model type."""
+        return DeepPot
+
+    def get_sel_type(self) -> List[int]:
+        """Get the selected atom types of this model."""
+        return []
+
+    def get_numb_dos(self) -> int:
+        """Get the number of DOS."""
+        return 0
+
+    def get_has_efield(self):
+        """Check if the model has efield."""
+        return False
+
+    def get_ntypes_spin(self):
+        """Get the number of spin atom types of this model."""
+        return 0
+
+    _OUTDEF_DP2PT: ClassVar[dict] = {
```

Comment:
should be placed in backend indepdent infer?

Reply:
Currently they have different names, atom_virial vs atomic_virial

Reply:
I would suggest unifying the names to **atomic_xxx**. what do you think?

Reply:
`o_atom_virial` has been hardcoded in all existing TF models, though.

Reply:
then we have to made them **atom_xxx***...

Reply:
https://github.com/deepmodeling/deepmd-kit/blob/b875ea8f6661b6e1567537ead7e2b4a8b14ea113/source/tests/infer/se_atten_no_atten_1.pbtxt#L12996-L12997

Reply:
> then we have to made them **atom_xxx***...

It will also affect PT models that have been saved. @CaRoLZhangxy also needs to change the name in #3162. Perhaps it should be conducted in a separated PR?

Reply:
we change the convention in the PT models. let make it in a separate PR.